tests/internal/test_module.py

```diff
@@ -339,3 +339,39 @@ def test_module_watchdog_dict_shallow_copy():
     assert original_modules == new_modules
 
     ModuleWatchdog.uninstall()
+
+
+@pytest.mark.skipif(sys.version_info < (3, 5), reason="requires python3.5 or higher")
```

Comment:
Can we expand on this reason? Why is python2.7 not supported

Reply:
[LazyLoader](https://docs.python.org/3/library/importlib.html#importlib.util.LazyLoader) was introduced in 3.5. I'll change the reason to mention this.